### PR TITLE
[sensorpush] Initial contribution

### PR DESCRIPTION
**Pull request by bobadair**
[SensorPush](https://www.sensorpush.com/) sells a line of battery-powered wireless sensors that, depending on the model, provide data on temperature, relative humidity, barometric pressure, dew point, and vapor pressure deficit (VPD). The sensors communicate using Bluetooth LE. While they can be used directly via Bluetooth, when multiple sensors are in use they are typically configured to relay data to a cloud service via the G1 WiFi Gateway. This binding retrieves sensor data from the SensorPush Gateway Cloud using a published API.

This is a replacement for PR #10310.

### CODE REVIEW

**Comment by lsiepel** [comment]
Did you fix all the review comments from the previous PR, like the textual config and oauth? Can you also fix the DCO?

File: bundles/org.openhab.binding.sensorpush/pom.xml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.openhab.addons.bundles</groupId>
+    <artifactId>org.openhab.addons.reactor.bundles</artifactId>
+    <version>4.1.0-SNAPSHOT</version>
```
Comment (wborn) [inline bundles/org.openhab.binding.sensorpush/pom.xml:10]:
```suggestion
    <version>4.2.0-SNAPSHOT</version>
```

**Comment by lsiepel** [comment]
@bobadair are you able to fix the DCO and build? I might review this PR, just checking if you are able to continue.

**Comment by lsiepel** [comment]
@bobadair As there has been some time without any updated / commits to this PR. Do you think you will be able to finish this or did you abandon this PR? At some time we will close this PR due to inactivity.

Or let me know if you need anything to finish this.

**Comment by lsiepel** [comment]
This binding seems to be a nice addition, so i hope it will get finsihed at some point in time. That being said as this binding PR has been here for a while without progress. So i close this for now.
Obviously the code is not gone and work can continue outside of this PR/repository. Once the work is picked up again, i'm happy to review any newly created PR.

**Pull request closed without merging**